util/OptionsDB.cpp

```diff
@@ -137,8 +137,7 @@ OptionsDB::OptionsDB() {
     s_options_db = this;
 }
 
-void OptionsDB::Commit()
-{
+void OptionsDB::Commit() {
```

Comment:
> Except style change is entirely what this last commit was about (what Geoff just asked for).
>
> This line is the main reason why you (and Geoff) had to keep pointing out style inconsistencies, because the original file (the example I had to go by) wasn't consistent.

The request was related to a specific changed line, not asking for any reformatting of the whole file.

> If you would prefer a new PR, entirely for moving the bracket on this line, I will do that.
> Or you can make an exception in this one particular case, since I was making a commit for style corrections in this file anyways.

I'm not interested in any of those, please undo that change.